src/internal/signalr/handler.go

```diff
@@ -247,17 +303,7 @@ func (h *observationsHandler) handleLifetimeEnergy(observation model.Observation
 
 	h.cache.SetLifetimeEnergy(val)
```

Comment:
I'd propose a different algorithm here:
* store the last accepted reading together with it's timestamp
* once there's a new incoming reading, truncate the timestamps of the incoming one and the last accepted one to full hours and check, if the new reading belongs to a new hour
* if it belongs to the next hour, start a goroutine, which will gather incoming readings with a 30-second timeout
* it the time is up, set the new reading in cache and send FIMP reports. The goroutine can be stopped at this point.